tt_eager/tt_lib/csrc/tt_lib_bindings_tensor_xary_ops.cpp

```diff
@@ -234,6 +234,13 @@ namespace tt::tt_metal::detail {
             R"doc(Perform an eltwise-modulus operation on ``{0}`` and ``{1}``. Formula : ``a - a.div(b, rounding_mode="floor") * b`` . Support provided only for WH_B0.)doc",
             R"doc("value", "float", "")doc"
 
+        );
+        detail::bind_unary_op_with_param(
+            m_tensor, "unary_fmod", unary_fmod,
+            py::arg("value"),
+            R"doc(Perform an eltwise-fmod operation on ``{0}`` and ``{1}``. Formula : ``a - a.div(b, rounding_mode="trunc") * b`` . Support provided only for WH_B0.)doc",
```

Comment:
Need to add the assertion for only WH.

Other question is if this supports integer dtype for in/out or not? Or else we need to add assert for this as well

Reply:
Ignore assertion for WH request, I see it was added.